.buildkite/pipeline_perf.py

```diff
@@ -98,7 +98,7 @@
     pytest_opts = ""
     if REVISION_A:
         devtool_opts += " --ab"
-        pytest_opts = f"{ab_opts} run {REVISION_A} {REVISION_B} --test {test_path}"
+        pytest_opts = f"{ab_opts} run build/{REVISION_A}/build/cargo_target/x86_64-unknown-linux-musl/release build/{REVISION_B}/build/cargo_target/x86_64-unknown-linux-musl/release --test {test_path}"
```

Comment:
should this be:

```
        pytest_opts = f"{ab_opts} run build/{REVISION_A}/build/cargo_target/$(uname -m)-unknown-linux-musl/release build/{REVISION_B}/build/cargo_target/$(uname -m)-unknown-linux-musl/release --test {test_path}"
```

to account for CPU architecture

Reply:
Ah, yeah, I guess, but this is deleted again in the next commit anyway, haha